api/provider/aws/environment_create.go

```diff
@@ -83,10 +95,10 @@ func (e *EnvironmentProvider) Create(req models.CreateEnvironmentRequest) (*mode
 	return e.Read(environmentID)
 }
 
-func (e *EnvironmentProvider) createSG(environmentID, vpcID string) (*ec2.SecurityGroup, error) {
+func (e *EnvironmentProvider) createSG(groupName, description, vpcID string) (*ec2.SecurityGroup, error) {
```

Comment:
This function is doing to much - we should try to make helper functions wrap a single aws api call. This should be split into `createSG` and `allowSGIngress`